4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-highlights-internationalization-live-preview.md

---
type: entry
section: highlights
rank: 4
importance: 2
anchor: internationalization-live-preview
title: Internationalization live preview
blockquote: Hello
text: |
  This new feature is key to our ideal of making gaming accessible for everyone.

>   From now on, developers will be able to preview translations directly from the editor viewport. This will ease up the burden of testing the GUI in multiple languages.

Reviewer: ```suggestion
  From now on, developers will be able to preview translations directly from the editor viewport. This will ease the burden of testing the GUI in multiple languages.
```

>   How does your new UI react being in French or Dutch? Or how does it look in simplified or traditional Chinese, or even in Hindi?

Reviewer: ```suggestion
  How does your new UI react to being in French or Dutch? Or how does it look in simplified or traditional Chinese, or even in Hindi?
```

>   [^internationalization-live-preview-layout-direction]: The layout direction only follows the main locale currently. [We're working on fixing this issue.](https://github.com/godotengine/godot/pull/97918)

Reviewer: ```suggestion
  [^internationalization-live-preview-layout-direction]: The layout direction only follows the main locale currently. [We’re working on fixing this issue.](https://github.com/godotengine/godot/pull/97918)
```

---
type: entry
section: highlights
rank: 4
importance: 2
anchor: internationalization-live-preview
title: Internationalization live preview
blockquote: Hello
text: |
  This new feature is key to our ideal of making gaming accessible for everyone.

  From now on, developers will be able to preview translations directly from the editor viewport.[^internationalization-live-preview-layout-direction] This will ease the burden of testing the GUI in multiple languages.

  How does your new UI react to being in French or Dutch? Or how does it look in simplified or traditional Chinese, or even in Hindi?

>   [^internationalization-live-preview-layout-direction]: The layout direction only follows the main locale currently. [We’re working on fixing this issue.](https://github.com/godotengine/godot/pull/97918)

Reviewer 2: ```suggestion
  [^internationalization-live-preview-layout-direction]: The layout direction only follows the main locale currently. [We’re working on fixing this issue.](https://github.com/godotengine/godot/pull/110378)
```

The old PR contained too many conflicts, I fixed them and submitted a more refined one ;)
